modules/thermal_hydraulics/src/userobjects/NumericalFlux1D.C

```diff
@@ -0,0 +1,109 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NumericalFlux1D.h"
+#include "THMUtils.h"
+
+InputParameters
+NumericalFlux1D::validParams()
+{
+  InputParameters params = GeneralUserObject::validParams();
+  return params;
+}
+
+NumericalFlux1D::NumericalFlux1D(const InputParameters & parameters)
+  : ThreadedGeneralUserObject(parameters),
+
+    _cached_flux_elem_id(libMesh::invalid_uint),
+    _cached_flux_side_id(libMesh::invalid_uint),
+
+    _last_region_index(0)
```

Comment:
who sets this

Reply:
ADNumericalFlux3EqnHLLC. It's used in unit testing to ensure that we hit the expected regions.